### ISSUE TITLE
use embedded resources instead of wwwroot

### ISSUE THREAD

**Issue by kgsensei** [tryphotino/photino.NET#194]
I'm checking out this project and it looks perfect for what I need, but I want to know if it's possible to load files from embedded resources instead of just from the /wwwroot directory?
I haven't actually used photino yet, so I apologize if this is obvious or redundant.

**Comment by Slluxx** [comment]
I would like to know this too. Its pretty much a deal breaker and any other solution i have looked at (Ultralight, Molybden, etc etc) all use external resources, which is against my idea of a portable app.

**Comment by CypherPotato** [comment]
It's not impossible, you can embed the content in your assembly and serve the files with an HTTP server built into the application where your Photino is, and listen to the server that is serving these files directly from the assembly. Photino already has it's own [HTTP server](https://github.com/tryphotino/photino.NET.Server), but I made an example using [Sisk](https://github.com/sisk-http/core) to serve that static content.

You can view the example I made [here](https://github.com/sisk-http/quickstart/blob/master/quickstart.photino-app/Program.cs).

Basically, you copy the reading stream from `Assembly.GetManifestResourceStream` to the output of the HTTP response. The example is very basic, it was poorly tested and I only included some supported MIME types. You must adjust according to your needs.

**Comment by Slluxx** [comment]
As per mentioned issue and templates, it should already be possible without any modifications because it already includes a static file server. The solution here is not to reinvent the wheel but to use and or fix what is already included with photino.
